modules/serverless-ossm-verifying-the-integration.adoc

```diff
@@ -0,0 +1,66 @@
+:_content-type: PROCEDURE
+[id="serverless-ossm-verifying-the-integration_{context}"]
+= Verifying the integration
+
+After installing {SMProductShortName} and {ServerlessProductShortName} with Istio enabled, you can verify that the integration works.
+
+.Procedure
+
+. Create a Knative Service that has sidecar injection enabled and uses a pass-through route:
++
+.Example `knative-service.yaml` configuration file
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  namespace: <namespace> <1>
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true" <2>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <3>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: <image_url>
+----
+<1> A namespace that is part of the service mesh member roll.
+<2> Instruct Knative Serving to generate an OpenShift pass-through enabled route, so that the certificates you have generated are served through the ingress gateway directly.
+<3> Inject {SMProductShortName} sidecars into the Knative service pods.
++
+[IMPORTANT]
+====
+You always need to add the annotation from this example to all your Knative Service to make them work with {SMProductShortName}.
```

Comment:
Rewrite to reduce wordiness: "Always add the annotation from this example to all of your Knative Service..."